### ISSUE TITLE
Crosswalk IDB failure 

### ISSUE THREAD

**Issue by nickcolley** [pouchdb/pouchdb#4307]
Reported in Slack:

Hi everyone,
Does anyone have a clue what would cause PouchDB not to respond to requests at all?
We’ve been using Pouch in production for a month or so, but certain devices running android crosswalk recently stopped working. A simple test like this never responds:

```
var cdb = new PouchDB('a_brand_new_db’);
cdb.get("a_non_existent_doc",function(data,err){console.log(data,err);});
```

throws

```
msg: "Failed to open indexedDB, are you in private browsing mode?"
this: IDBOpenDBRequest
error: DOMError
message: "Version change transaction was aborted in upgradeneeded event handler."
name: “AbortError”
```

**Comment by nolanlawson** [comment]
Another update: 

Regarding my previous report and #4307, looks like Nolan was right about us exceeding quota of ~10%.   Not sure how we ended up with above error, but possibly due to upgrading and downgrading xwalk in the middle of this mess...

Something worth noting  is that I got mixed results upon exceeding quota. With WebSQL, every time I got 500 ""there was not enough remaining storage space…”, but with idb, no exceptions are thrown and neither my .then or .catch is called. I exceeded quota while a view was building, on both idb and websql.
